conda-recipes/llvmlite/meta.yaml

```
@@ -30,7 +30,8 @@ requirements:
   host:
     - python
     # On channel https://anaconda.org/numba/
-    - llvmdev 11.1.0 *4 # [not win]
+    - llvmdev 11.1.0 *5 # [(osx and arm64)]
+    - llvmdev 11.1.0 *4 # [not (osx and arm64) and not win]
```

```suggestion
    - llvmdev 11.1.0 *4 # [not ((osx and arm64) or win)]
```
think this is a bit easier to parse (assuming it works)?